internal/lens/fetcher.go

```diff
@@ -98,7 +98,7 @@ func (f *lensedFetcher) Init(
 
 historyLoop:
 	for _, historyItem := range history {
-		sources := historyItem.collection.CollectionSources()
+		sources := historyItem.collection.VersionSources
```

Comment:
question: Not related to this specific line but it made me think about the query sources's transforms. Just wondering why they aren't taken into account.

Reply:
`CollectionSources()`, and it's counterpart, `QuerySources()` filtered the backing slice by type.

Reply:
yes